src/tool_source/github.rs

```diff
@@ -18,20 +19,24 @@ const APP_NAME: &str = "LPGhatguy/aftman";
 
 pub struct GitHubSource {
     client: Client,
+    token: Option<String>,
 }
 
 impl GitHubSource {
-    pub fn new() -> Self {
+    pub fn new(auth: Option<AuthManifest>) -> Self {
         Self {
             client: Client::new(),
+            token: auth.map(|t| t.github).flatten()
```

Comment:
Related to previous comment

```suggestion
            token: auth,
```